3.2/root/usr/share/container-scripts/mongodb/init-petset-replset.sh

```diff
@@ -124,7 +124,7 @@ function add_member() {
     return 1
   fi
 
-  info "Successfully joined replica set"
+  info "Successfully connected to replica set"
```

Comment:
For the history: initial fix was about removing two identical messages about joining to replica set. After quick conversion offline we found a way of making these messages better than before.